paper-card.html

```diff
@@ -117,7 +117,7 @@
     </style>
 
     <div class="header">
-      <img hidden$="[[!image]]" src="[[image]]">
+      <iron-image hidden$="[[!image]]" src="[[image]]" sizing="fit-width" width="100%" preload fade></iron-image>
```

Comment:
I thiiiiiink like in https://github.com/PolymerElements/paper-card/pull/31 it would be nice if we could have preload and fade configurable